docs/ops/arithmetic/Add_1.md

```diff
@@ -4,48 +4,50 @@
 
 **Category**: Arithmetic binary operation
 
-**Short description**: *Add* performs element-wise addition operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *Add* performs element-wise addition operation with two given tensors applying broadcasting rule specified in the *auto_broacast* attribute.
+
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attribute is not `none`. Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *Add* performs addition operation for the input tensors *a* and *b* using the formula below:
+
+\f[
+o_{i} = a_{i} + b_{i}
+\f]
 
 **Attributes**:
 
 * *auto_broadcast*
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+    * *none* - no auto-broadcasting is allowed, all input shapes must match,
+    * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
```

Comment:
So, have we decided to hide PDPD rule?

Reply:
@slyalin, shall we support PDPD as one of auto_broadcast attributes for Add?

Reply:
PDPD type is added as supported broadcast type by Add.